src/uct/ib/ud/base/ud_ep.c

```diff
@@ -493,26 +493,30 @@ static ucs_status_t uct_ud_ep_disconnect_from_iface(uct_ep_h tl_ep)
     return UCS_OK;
 }
 
-ucs_status_t uct_ud_ep_create_connected_common(uct_ud_iface_t *iface,
-                                               const uct_ib_address_t *ib_addr,
-                                               const uct_ud_iface_addr_t *if_addr,
-                                               unsigned path_index,
-                                               uct_ud_ep_t **new_ep_p,
-                                               uct_ud_send_skb_t **skb_p)
+ucs_status_t uct_ud_ep_create_connected(uct_iface_h iface_h,
```

Comment:
1. why removed "common"
2. maybe pass "params" instead of multiple args?
3. rename iface_h -> tl_iface or uct_iface

Reply:
done